server/service/src/vaccination/update/generate.rs

```diff
@@ -94,6 +94,7 @@ pub fn generate(
                     reason_id: None,
                     note: None,
                 }],
+                status: Some(InsertCustomerReturnStatus::New),
```

Comment:
Passing status as New here (it must be new to add lines)

This is subsequently updated to verified in a second DB call.